libyul/AsmParser.cpp

```diff
@@ -304,9 +304,6 @@ std::map<evmasm::Instruction, string> const& Parser::instructionNames()
 	{
```

Comment:
Turns out this `instructionNames` function is not even used anymore, I guess since `EVMDialect`?

Reply:
Please remove it.

Reply:
Done.